### PR TITLE
WRO-12241: Update unit test for packages/core/internal files

### PR DESCRIPTION
**Pull request by 0x64**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Unit tests could be added to packages/core/internal files for quality management.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Added unit tests to cover more cases.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-12241

### Comments
Enact-DCO-1.0-Signed-off-by: Seungcheon Baek (sc.baek@lge.com)

### CODE REVIEW

File: packages/core/internal/prop-types/tests/prop-types-specs.js [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+
+			expect(consoleErrorMock).not.toHaveBeenCalled();
+		});
+
+		test('should call console.error for EnactPropTypes.ref if ref value is given', () => {
+			const ref = dummyString;
+
+			render(
+				<TestComponent
+					typeRef={ref}
```
Comment (juwonjeong) [inline packages/core/internal/prop-types/tests/prop-types-specs.js:135]:
```suggestion
					typeRef={dummyString}
```

File: packages/core/internal/prop-types/tests/prop-types-specs.js [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+				/>
+			);
+
+			expect(consoleErrorMock).not.toHaveBeenCalled();
+		});
+
+		test('should call console.error for EnactPropTypes.ref if ref value is given', () => {
```
[the remark this answers is not in the document]
Comment (0x64) [reply packages/core/internal/prop-types/tests/prop-types-specs.js]: Thanks for catching. I changed to 'non-ref' as the testing purpose.

File: packages/core/internal/prop-types/tests/prop-types-specs.js (hunk 1 above)
Comment (0x64) [reply packages/core/internal/prop-types/tests/prop-types-specs.js]: Updated. Also removed a useless const definition.

**Review by juwonjeong — APPROVED** [review]
LGTM

**Pull request merged**